src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1682,32 +1657,19 @@ private void DoCreateRunspace(string initialCommand, bool skipProfiles, bool sta
             // Record how long it took from process start to runspace open for telemetry.
             _readyForInputTimeInMS = (DateTime.Now - Process.GetCurrentProcess().StartTime).TotalMilliseconds;
 
-            DoRunspaceInitialization(importSystemModules, skipProfiles, initialCommand, configurationName, initialCommandArgs);
+            DoRunspaceInitialization(skipProfiles, initialCommand, configurationName, initialCommandArgs);
         }
 
-        private void OpenConsoleRunspace(Runspace runspace, bool staMode)
+        private void OpenConsoleRunspace(Runspace runspace)
         {
-            // staMode will have following values:
-            // On FullPS: 'true'/'false' = default('true'=STA) + possibility of overload through cmdline parameter '-mta'
-            // On NanoPS: always 'false' = default('false'=MTA) + NO possibility of overload through cmdline parameter '-mta'
-            // ThreadOptions should match on FullPS and NanoPS for corresponding ApartmentStates.
-            if (staMode)
-            {
-                // we can't change ApartmentStates on CoreCLR
-#if !CORECLR
-                runspace.ApartmentState = ApartmentState.STA;
-#endif
-                runspace.ThreadOptions = PSThreadOptions.ReuseThread;
-            }
```

Comment:
Is this removal intentional? I think if that's not set, we create a new thread every time you execute a new command line.

Reply:
Will put back the ReuseThread line

Reply:
Actually, this line was only within the `staMode` which was not enabled for the user to set.  So currently, we use the default which is to use a new thread for each pipeline.  Do we want to change this behavior?

Reply:
`staMode = true` was the default even if the user couldn't set it, right?
At any rate, I do think we want to reuse the thread.

Reply:
No, we defaulted to MTA in PSCore

Reply:
I see. We defaulted to STA on Windows for many years, so reusing the thread seems preferable.

Reply:
@SteveL-MSFT - are you putting it back?

Reply:
I need to verify something on Windows (was using a Mac for the changes) before I put it back.  With my other uber PR to remove apartmentstate all-up, the WSMan provider didn't work correctly.

Reply:
Looks good on my Windows box, will run through the tests and push the change.